api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -141,6 +141,20 @@ public String getDisplayName() {
         else            return getTypeDisplayName();
```

Comment:
If it is an override then we cannot deprecate it. We *could* consider moving the REST-exported property elsewhere, or just leave it, although it is kind of useless.